Assets/Tests/Runtime/GameEvents/GameEventTest.cs

```diff
@@ -122,6 +123,8 @@ public void ShouldRaiseGameEventEvent()
         [Test]
         public void ShouldNotBreakChainWhenExceptionIsThrown()
         {
+            LogAssert.ignoreFailingMessages = true;
```

Comment:
@KacperKenjiLesniak this was causing issues previously in regards to failing test. What a bad design from Unity side :V

Reply:
Well... Glad it works now :D